### PR TITLE
Document how to curate datasets for loading where place curation files

### LINKED ISSUE #17: Document how to curate datasets for loading & where place curation files
doc should be in Markdown format in the doc folder on this repo

### PR DESCRIPTION
**Pull request by ubyndr**
Resolves #17 

cc @JABelfiore

### CODE REVIEW

**Comment by ubyndr** [comment]
Hi @JABelfiore  and @dosumis ,

Could you please take a look and review the `dataset_curation_guidelines.md` file I’ve created?

Thank you!

File: docs/dataset_curation_guidelines.md [hunk 1]
```diff
@@ -0,0 +1,141 @@
+# Dataset Curation Guidelines for Loading and File Placement
+
+This guide outlines the process for curating datasets and specifies where to place the curation
```
Comment (JABelfiore) [inline docs/dataset_curation_guidelines.md:3]: Suggest amend to "and specifies the location where curation files should be placed for proper loading"

File: docs/dataset_curation_guidelines.md [hunk 2]
```diff
@@ -0,0 +1,141 @@
...
+    - **Example**: "Cell types"
+    
+11. **Value type(s):**
+    - **Description**: This column specifies if the values in the dataset are represented in full 
+      names or as abbreviations.
+    - **Example**: "abbreviations"
+    
+12. **Notes:**
+    - **Description**: Any additional notes or comments regarding the dataset.
+    - **Example**: "Muto et al. (2021) Nat Commun"
```
Comment (JABelfiore) [inline docs/dataset_curation_guidelines.md:79]: Example here should be notes on dataset not publication title

File: docs/dataset_curation_guidelines.md [hunk 3]
```diff
@@ -0,0 +1,141 @@
...
+13. **Study Short Name:**
+    - **Description**: The shortened name or acronym of the study associated with the dataset.
+    - **Example**: "Muto et al. (2021) Nat Commun"
+    
+14. **CxG Dataset Collection X:**
+    - **Description**: The CellxGene link to the collection where the dataset is stored.
+    - **Example**: "https://cellxgene.cziscience.com/collections/9b02383a-9358-4f0f-9795-a891ec523bcc"
+    
+15. **Is the dataset Normal or Normal/Diseased:**
+    - **Description**: Indicates whether the dataset is of normal samples or includes both
```
Comment (JABelfiore) [inline docs/dataset_curation_guidelines.md:90]: Suggest amend: "Indicates whether the dataset includes normal samples, diseased samples or both"

**Review by JABelfiore — COMMENTED** [review]
Thanks for this! added a couple inline comments

**Pull request merged**